Source/Core/Core/PowerPC/MMU.h

```diff
@@ -49,7 +49,15 @@ struct TryReadResult
   // the actual value that was read
   T value;
 
+#ifdef _MSC_VER
+#pragma warning(push)
+// Disable warning for uninitialized member variables
```

Comment:
Can you add a reason _why_ this warning is temporarily disabled for this ctor? (Or: simply address it, so the suppression isn't needed)

Reply:
It's because this class is kinda like `std::optional`.  Probably a better solution would be to replace it with `std::optional` to guarantee nobody can touch the other fields.

Reply:
I've replaced `TryReadResult` and `TryWriteResult` with `std::optional<ReadResult>` and `std::optional<WriteResult>`.